marlowe-playground-client/src/MonadApp.purs

```diff
@@ -278,9 +278,13 @@ updatePossibleActions oldState =
   addButPreserveActionInputs oldInputs actionInputIdx m actionInput = appendValue m oldInputs (actionPerson actionInput) actionInputIdx actionInput
 
   actionPerson :: ActionInput -> (Maybe PubKey)
```

Comment:
Slightly confusing because this is actually getting `Maybe String` and that `String` can be either a `PubKey` or `TokenName` which are both just aliases for `String`. I don't think we should change anything, just pointing out.